MetaMorpheus/GUI/Views/SaveSettingsWindow.xaml

```diff
@@ -0,0 +1,33 @@
+﻿<Window x:Class="MetaMorpheusGUI.SaveSettingsWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:MetaMorpheusGUI"
+        xmlns:guiFunctions="clr-namespace:GuiFunctions;assembly=GuiFunctions"
+        mc:Ignorable="d"
+        Title="SaveSettingsWindow" Height="135" Width="600">
+
+
+    <Grid >
+        <Grid.RowDefinitions>
+            <RowDefinition Height="2*"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+
+        <TextBox Grid.Row="0" Text="{Binding TypedSettingsName}" Height="35"
+                 HorizontalAlignment="Stretch"  VerticalAlignment="Top"
+                 Margin="10" FontSize="25" x:Name="NameTextBox">
+        </TextBox>
+
+        <Button x:Name="Cancel"  Grid.Row="2" Content="Cancel" 
+                FontSize="20" Height="30" IsCancel="true" Click="CloseWindow"
+                Width="200" VerticalAlignment="Center"  HorizontalAlignment="Right"/>
+        
+        <Button x:Name="Save" Grid.Row="2" Content="Save" FontSize="20" Height="30" 
+                Width="200" VerticalAlignment="Center" HorizontalAlignment="Left"
+                 Click="Save_Click"/>
+        <!--Command="{Binding SaveSettingsFromWindowCommand}" 
+                CommandParameter="{Binding TypedSettingsName}"-->
```

Comment:
We should be able to bind a command to the save button instead of routing through an event Save_Click